src/github.com/getlantern/flashlight/logging/error_aggregator.go

```diff
@@ -0,0 +1,176 @@
+package logging
```

Comment:
Aggregating errors is tricky as different error has different set of fields available. Error volume should be very low in normal time, but will burst when there's connectivity issue or server is blocked. Rate limiting may be more useful in that case.